modules/web-console/web-console-server/src/test/java/org/apache/ignite/console/web/security/AuthenticationTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.console.web.security;
+
+import java.util.UUID;
+import org.apache.ignite.console.AbstractSelfTest;
+import org.apache.ignite.console.dto.Account;
+import org.apache.ignite.console.repositories.AccountsRepository;
+import org.apache.ignite.internal.util.typedef.internal.U;
+import org.apache.ignite.testframework.GridTestUtils;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.LockedException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+/**
+ * Tests for authentication.
+ */
+@SpringBootTest(properties = {"account.authentication.maxAttempts=20"})
+public class AuthenticationTest extends AbstractSelfTest {
```

Comment:
Add tests when user try to authenticate after when lock time was expired.

Reply:
Extend test